url.bs

```diff
@@ -119,9 +119,12 @@ error.
 <h3 id=percent-encoded-bytes>Percent-encoded bytes</h3>
 
 <p>A <dfn>percent-encoded byte</dfn> is U+0025 (%), followed by two <a>ASCII hex digits</a>.
-Sequences of <a lt="percent-encoded byte">percent-encoded bytes</a>,
-<a for=string>percent-decoded</a>, should not cause <a>UTF-8 decode without BOM or fail</a> to
-return failure.
+
+<p class=note>It is generally a good idea for sequences of <a>percent-encoded bytes</a>, when
+<a for=string>percent-decoded</a> and then passed to <a>UTF-8 decode without BOM or fail</a>, to not
+end up as failure. How important this is depends on where <a>percent-encoded bytes</a> are used,
+e.g., for the <a>host parser</a> it is fatal, whereas for
+<a href="#url-rendering-i18n">URL rendering</a> it is merely beneficial.
```

Comment:
Maybe "for the host parser such sequences will cause failure, whereas for URL rendering [fill in the consequences or lack thereof here]"